generate/function.go

```diff
@@ -0,0 +1,159 @@
+package generate
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/progrium/darwinkit/generate/codegen"
+	"github.com/progrium/darwinkit/generate/modules"
+	"github.com/progrium/darwinkit/generate/typing"
+)
+
+// this keeps track of types that we know we can't handle.
+var unhandledStructTypes = map[string]bool{
+	"CFAllocatorContext":                true,
+	"CFArrayCallBacks":                  true,
```

Comment:
i think we have most of these, for example:
https://github.com/progrium/darwinkit/blob/main/macos/corefoundation/corefoundation_structs.go#L232

Reply:
Gotcha, I should have cleaned this list up, will do.